### DISCUSSION TITLE
Overriding dropdown for more than 5 select objects

### DISCUSSION THREAD

**Discussion by mfein2** [datapane/datapane#164] [Help]
So I'm using a select statement with 5 blocks which means the blocks are listed in a dropdown menu instead of being displayed as tabs.  I keep trying to use the parameter type=dp.Select.TABS so I can see them as tabs and when I do I receive the error: "AttributeError: type object 'Select' has no attribute 'TABS'".

Where exactly should this parameter be entered?

Any help is appreciated!

**Comment by johnmicahreid** [comment]
Hi @mfein2,

Took a look and realized it was an error with our documentation - the parameter should be `type=dp.SelectType.TABS`. 

Sorry about that - I've updated the documentation now and want to thank you for reporting this!
